dex-it/src/test/scala/com/wavesplatform/it/sync/MultipleMatchersTestSuite.scala

```diff
@@ -67,6 +73,62 @@ class MultipleMatchersTestSuite extends MatcherSuiteBase {
     dex1.connectToNetwork()
   }
 
+  "WS Order book state should be the same on two matchers" in {
+    val acc = mkAccountWithBalance(100.eth -> eth, 100.waves -> Waves)
+
+    val wsob1 = mkWsOrderBookConnection(ethWavesPair, dex1)
+    val wsob2 = mkWsOrderBookConnection(ethWavesPair, dex2)
+
+    val sell = mkOrder(acc, ethWavesPair, SELL, 10.eth, 1.waves, 0.003.waves)
+    dex1.api.place(sell)
+
+    List(
+      mkOrder(alice, ethWavesPair, BUY, 5.eth, 1.waves, 0.003.waves),
+      mkOrder(alice, ethWavesPair, BUY, 3.eth, 1.waves, 0.003.waves),
+      mkOrder(alice, ethWavesPair, BUY, 2.eth, 1.waves, 0.003.waves)
+    ).foreach(dex1.api.place)
+
+    dex1.api.waitForOrderStatus(sell, Status.Filled)
+
+    eventually {
+      val obs1 = wsob1.receiveAtLeastN[WsOrderBookChanges](1).reduce((m1, m2) => mergeOrderBookChanges(m1, m2))
```

Comment:
Probably:
```
val obs1 = wsob1.receiveAtLeastN[WsOrderBookChanges](1).reduce(mergeOrderBookChanges)
```
is enough. Did you try? (in other places too)

Reply:
its enough, tnx